applications/form_specs.py

```diff
@@ -113,6 +113,11 @@
                         help_text=snippet("study_purpose-help_text"),
                         template_name="form_textarea",
                     ),
+                    Field(
+                        name="read_analytic_methods_policy",
+                        label="Have all applicants for OpenSAFELY read and agreed to the OpenSAFELY Analytic Methods Policy?",
```

Comment:
```suggestion
                        label="Have all applicants read and agreed to the OpenSAFELY Analytic Methods Policy?",
```

I think the "for OpenSAFELY" feels implicit here, but I'm aware there's been a bit of wrangling over the wording already so happy for you to disregard!

Reply:
Sadly, this isn't negotiable. I share your snag, however.